spec/checkPullRequestReviewSpec.js

```diff
@@ -1522,5 +1522,23 @@ describe('Pull Request Review Module', () => {
         });
       }
     );
+    describe(
+      'when author asks some question/comment and asks reviewer to ptal',
+      () => {
+        const initialCommentBody = commentPayloadData.payload.comment.body;
+        beforeAll(() => {
+          commentPayloadData.payload.comment.body =
+          'Hi I have a doubt @reviewer PTAL';
+        });
+        beforeEach(async () => {
+          await robot.receive(commentPayloadData);
+        });
+        it('should not comment on pull request', () => {
+          expect(github.issues.createComment).not.toHaveBeenCalled();
```

Comment:
I was expecting `Hi I have a doubt @reviewer PTAL` to trigger bot to assign @reviewer. [@jameesjohn Can you confirm this and check how this test passed here?]

And through issue #174 I meant a message like  `{{Question/comment}} @{{reviewer_username}} PTAL` should not trigger the bot. [Note: {{}} here are not meant for variable values i.e, not for interpolation]

Reply:
@DubeySandeep I updated the above pattern it passed with this one too

Reply:
@mridul-netizen, I think @DubeySandeep meant a message following that format. Not the exact text.

Reply:
Can you please suggest changes needed to be made?